src/site/layouts/page.drupal.liquid

```diff
@@ -73,6 +73,8 @@
         </div>
       </div>
     </main>
+    <!-- Up To Top Button -->
+    {% include "src/site/components/up_to_top_button.html" %}
```

Comment:
Won't this add the back-to-top button for all articles of type `page`? Meaning not just translated pages, but everything under the benefit hubs

Reply:
yes absolutely, good catch. went down the rabbit hole of trying to convert the `src/site/components/up_to_top_button.html` to jsx / a react component but then realized i could use this liquid filter. it would be nice if the up to top button is also available for use in React as well